docs/ci-cd/testing/acceptance-tests.md

```diff
@@ -219,6 +219,55 @@ and then run codeception with the `--debug` flag:
 vendor/bin/codecept run --debug tests/acceptance/YourTestCept.php
 ```
 
+### Using Gherkin
```

Comment:
This page is [synchronized](https://github.com/greenpeace/planet4-docs/blob/master/sync-repos.sh) from the [docker-compose repo](https://github.com/greenpeace/planet4-docker-compose/blob/master/docs/testing.md), so we make changes in one place. Can you open a PR for that there?

Reply:
🙇 ➡️ https://github.com/greenpeace/planet4-docker-compose/pull/51
Changed this PR to contain only the modification on `ci-cd/test-environments.md`